certificate/scipy_template_2016/template_SPC2016Acertificate

```diff
@@ -0,0 +1,21 @@
+* certificate template
```

Comment:
In scipy_template_2016  there are two copies of fossee logo checked in,
even for Kannan Sir signs, there are two entries.
Kindly remove duplication.